components/camel-tika/src/main/java/org/apache/camel/component/tika/TikaComponent.java

```diff
@@ -38,6 +38,10 @@ protected Endpoint createEndpoint(String uri, String remaining, Map<String, Obje
             tikaConfiguration.setTikaConfig(config);
         }
         tikaConfiguration.setOperation(new URI(uri).getHost());

```

Comment:
I just realized that `setProperties` should be done on the endpoint not on the configuration in order to avoid reflection, is that correct @oscerd ?

Reply:
Yes, we missed that one, I guess @davsclaus

Reply:
Yes see the other components how its done

Reply:
@omarsmak I've changed call of setProperties to invoked on endpoint (other components do it that way), is it correct now?